01/README.md

```diff
@@ -0,0 +1,251 @@
+# 01 - Introduction to Quarkus
+
+## What is Quarkus?
+
+Quarkus is an open-source Kubernetes-native Java framework tailored for GraalVM and HotSpot, crafted from best-of-breed Java  libraries and standards. The goal is to make Java the leading platform in Kubernetes and serverless environments while offering developers a framework to address a wider range of distributed application architectures.
+
+## How is Quarkus different to Spring boot
+
+### Spring Boot
+
+- Mature framework with a large community. Focus on microservices and enterprise applications.
+- Accelerates developer productivity with production-ready integrations. 
+- Reduces configuration and boilerplate through convention over configuration. 
+- Supports both blocking and non-blocking web capabilities but not simultaneously.
+
+### Quarkus
+
+- Similar to Spring Boot but emphasizes fast boot time, smaller memory footprint, and lower resource usage.
+- Optimized for cloud, serverless, and containerized environments.
+- Integrates well with popular Java frameworks.
+- Offers simultaneous both blocking and non-blocking strategies.
+- Offers Dev UI and live reload for rapid development and testing.
+- Offers a unified reactive API for both imperative and reactive programming styles.
+- Offers Native Images for even faster startup and lower memory footprint.
+
+## Quarkus Technologies
+
+- **Maven** to create a new project, add or remove extensions, launch development mode, debug your application, and build your application.
+- **Jakarta EE** to write your business logic.
+- **MicroProfile** to write your REST endpoints, inject configuration, and more.
```

Comment:
REST endpoints are actually form Jakarta EE (JAX-RS/Jakarta REST). MP is config, health, tracing, ...